osgp/platform/osgp-adapter-ws-smartmetering/src/test/java/org/opensmartgridplatform/adapter/ws/smartmetering/application/mapping/PushSetupAlarmMappingTest.java

```diff
@@ -42,30 +54,55 @@ public void testPushSetupAlarmMapping() {
                 org.opensmartgridplatform.domain.core.valueobjects.smartmetering.PushSetupAlarm
                     .class);
 
-    // check mapping
     assertThat(pushSetupAlarmMapped).isNotNull();
+
+    // check mapping of SendDestinationAndMethod
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod()).isNotNull();
     assertThat(pushSetupAlarmMapped.getSendDestinationAndMethod().getDestination()).isNotNull();
```

Comment:
Is this `isNotNull` check necessary as the destination is checked against an actual value later on?

Reply:
Removed all isNotNull checks. If it is null, checks later on will fail.